workflow/airflow_libero/dags/make_assets_public_dag.py

```diff
@@ -0,0 +1,238 @@
+from datetime import timedelta
+import os
+import json
+from typing import Dict
+
+import airflow
+from airflow import DAG
+from bs4 import BeautifulSoup
+from lxml.etree import (
+    Element,
+    SubElement,
+    fromstring,
+    tostring,
+)
+import requests
+
+from airflow.operators import EventEmittingPythonOperator
+
+PUBLIC_DATA_DIR = '/usr/local/airflow/public/articles'
+
+SUCCESS = True
+FAILURE = False
+STATIC_ASSETS_URL = 'http://localhost:8089'
+SUPPORTED_MEDIA_TYPES = ['image/tiff', "image/jpeg"]
+ASSET_ELEMENTS = ['source', 'variant']
+
+
+def convert_to_xml(article_content: Dict[str, Dict]) -> str:
+    """
+    <root>
+        <content-list>
+            <list-item>
+            </list-item>
+        </content-list>
+    </root>
+
+    :param article_content:
+    :return:
+    """
+    root = Element('root')
+    content_list = SubElement(root, 'content-list')
+
+    for content_item in article_content['content_items']:
+        list_item = SubElement(content_list, 'list-item')
+
+        name = SubElement(list_item, 'name')
+        name.text = content_item['name']
+
+        language = SubElement(list_item, 'language')
+        language.text = content_item['language']
+
+        model = SubElement(list_item, 'model')
+        model.text = content_item['model']
+
+        content = SubElement(list_item, 'content')
+        content.append(fromstring(content_item['content'].encode('utf-8')))
+
+    return tostring(root)
+
+
+def store_article_data(*args, **kwargs) -> bool:
+    """Store article data from input_data provided
+
+    :return: bool
+    """
+
+    input_data = kwargs['dag_run'].conf.get('input_data')
+
+    if input_data:
+        kwargs['ti'].xcom_push('run_id', input_data.get('run_id'))
+        kwargs['ti'].xcom_push('article_id', input_data.get('article_id'))
+        kwargs['ti'].xcom_push('article_version', input_data.get('article_version'))
+        kwargs['ti'].xcom_push('article_version_id', input_data.get('article_version_id'))
+        return SUCCESS
+
+    return FAILURE
+
+
+def fetch_article_content(*args, **kwargs) -> bool:
+    article_version_id = kwargs['ti'].xcom_pull(task_ids=None, key='article_version_id')
+    url = f'http://article-store:8000/articles/api/v1/article-versions/{article_version_id}/'
```

Comment:
`article-store:8000` should be configuration.

Reply:
There are a few values that should be provided as configuration here. Was part of a refactor that hasn't happened.

Will provide them as environment variables in the container to match other settings.